### DISCUSSION TITLE
Opschalingsanalyse vraag 4: Welke kosten en besparingen voor de gemeentelijke uitvoering zijn aan het sensorenregister verbonden?

### DISCUSSION THREAD

**Discussion by marcvanandel** [kadaster-labs/sensrnet-home#254] [Pilot Gemeente]
Opschalingsanalyse vraag 4: **Welke kosten en besparingen voor de gemeentelijke uitvoering zijn aan het sensorenregister verbonden?**

**Comment by WOostdam** [comment]
Kosten: Afhankelijk van het aantal sensoren (groei IoT) mogelijk 1 Fte voor bijhouden registratie. Kosten voor in stand houden omgeving waarop applicatie draait (beheerkosten en licentiekosten).
Baten: Meestal indirect. Voorkomen onnodige aanschaf sensoren door bestaande ook voor andere doeleinden te gebruiken (geen twee camera's op dezelfde plek, maar beelden delen). Kunnen verantwoorden waarom je waar een bepaalde sensor hebt geplaatst (schadeclaims en of boetes voorkomen).

**Reply by mikegroels** [reply]
Extra indirecte baat: Verrommeling van de buitenruimte tegengaan, doordat plaatsers van sensoren (devices) zien wat er zoal in de buurt hangt.

**Comment by marcvanandel** [comment]
Door een gezamenlijk product te ontwikkelen worden de gezamenlijke kosten gedrukt. Bovendien wordt er uniformer data ingewonnen.

Hoe kan dit als besparing worden uitgedrukt?

**Reply by marcvanandel** [reply]
Uniforme data zou meer duidelijkheid moeten geven bij aanbestedingen wat een besparing moet betekenen (en toch moeilijk in geld uit te drukken)

**Reply by marcvanandel** [reply]
Ontwikkelen van pakketten kan ook goedkoper omdat duidelijk is wat er precies geleverd moet worden (besparing bij potentiële leveranciers van sensoren registratie tools)

**Comment by marcvanandel** [comment]
Naast geld / harde euro's zijn er ook **kwalitatieve baten**:
- transparantie naar de burger -> moeilijk in geld uit te drukken
- tegengaan van verrommeling -> voorkomen van aanschaf dubbele sensoren, dubbel beheer van informatie die zijn nog wel enigszins in euro's uit te drukken. Het minder 'vervuilen' van de (zichtbare) openbare ruimte is een waarde die moeilijk in geld is uit te drukken
- uniformiteit en beschikbaar hebben van metadata -> biedt mogelijkheden ... die we nog niet goed overzien en kunnen uitdrukken in geld

**Comment by marcvanandel** [comment]
Besparingen komt ook door Common Ground in brede zin. Door meer Common Ground tools te ontwikkelen en beschikbaar te hebben, dragen we bij aan de common ground waar alle gemeenten steeds meer op aan kunnen sluiten. Dat geeft een rijkere set aan tools die beschikbaar zijn en tegelijk goedkoper door deze strategie.
